pkg/tfbridge/tokens_test.go

```diff
@@ -506,6 +510,7 @@ func testTokenAliasing(t *testing.T) {
 	}
 
 	simple := provider()
+	simple.Version = "1.0.0"
```

Comment:
This test is weird, do we have a bug in the prod version of aliasing or am I missing something obvious? I think that without this addition, we're going from version 0 to version 1.0.0. In this case of a major version change the policy is to stop doing RenameResourceWithAlias and simply add to r.Aliases, and the new implementation does that but the asserts are not happy. Is version 0 special?